.github/workflows/test_attestation.yml

```diff
@@ -58,15 +61,36 @@ jobs:
         env:
           SIDECAR_REGISTRY: ${{ inputs.sidecar_registry }}
           ATTESTATION_SIDECAR_IMAGE: ${{ inputs.attestation-image }}
+          REPO_DISPATCH_TAG: ${{ github.event.client_payload.image_tag }}
         run: |
           if [ -z "$SIDECAR_REGISTRY" ]; then
             SIDECAR_REGISTRY="http://confidentialsidecars.azurecr.io"
           fi
-          if [ -z "$ATTESTATION_SIDECAR_IMAGE" ]; then
+          if [ -n "$REPO_DISPATCH_TAG" ]; then
+            ATTESTATION_SIDECAR_IMAGE="attestation:$REPO_DISPATCH_TAG"
+          elif [ -z "$ATTESTATION_SIDECAR_IMAGE" ]; then
             ATTESTATION_SIDECAR_IMAGE="attestation:main"
           fi
           echo "manifest-modifiers=ATTESTATION_SIDECAR_IMAGE=$SIDECAR_REGISTRY/$ATTESTATION_SIDECAR_IMAGE" >> $GITHUB_OUTPUT
 
+  pipeline-link-comment:
+    name: Create Pipeline Link Comment in PR
+    if: github.event_name == 'repository_dispatch'
+    runs-on: ubuntu-latest
+    env:
+      RUN_ID: ${{ github.run_id }}
+      PR_NUMBER: ${{ github.event.client_payload.pr_number }}
+    steps:
+      - name: Create Pipeline Link Repo Dispatch Request
+        run: |
+          curl -L \
```

Comment:
same comment from the other PR about adding the --fail flag to `curl`
https://github.com/microsoft/confidential-sidecar-containers/pull/80#discussion_r1472008543